shell/common/shell.cc

```diff
@@ -642,12 +642,14 @@ bool Shell::Setup(std::unique_ptr<PlatformView> platform_view,
   weak_platform_view_ = platform_view_->GetWeakPtr();
 
   // Setup the time-consuming default font manager right after engine created.
-  fml::TaskRunner::RunNowOrPostTask(task_runners_.GetUITaskRunner(),
-                                    [engine = weak_engine_] {
-                                      if (engine) {
-                                        engine->SetupDefaultFontManager();
-                                      }
-                                    });
+  if (!settings_.prefetched_default_font_manager) {
```

Comment:
nit: consider updating the comment above to explain the guard, and where we do a last minute check to make sure it's set up.

Also consider documenting that we do it this way because it's guarded by `SkOnce`, so we can be confident a second call will be very quick as long as the first call has finished.